common/views/components/ButtonInline/ButtonInline.tsx

```diff
@@ -93,6 +93,4 @@ const ButtonInline = forwardRef<HTMLButtonElement, ButtonInlineProps>(
   }
 );
 
-ButtonInline.displayName = 'ButtonInline';
```

Comment:
<img width="799" alt="Screenshot 2022-03-07 at 16 17 57" src="https://user-images.githubusercontent.com/1394592/157073978-005658f5-5eb5-489e-8b4b-01b52d1362d7.png">

Hands up! It's the lint police.